KizGetdown.html

```diff
@@ -0,0 +1,121 @@
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarNav"
+          aria-controls="navbarNav"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item px-3">
+              <a class="nav-link" href="index.html"
+                >Home <span class="sr-only">(current)</span></a
+              >
+            </li>
+            <li class="nav-item active px-3">
+              <a class="nav-link" href="KizGetdown.html">KizGetdown</a>
+            </li>
+            <li class="nav-item px-3">
+              <a class="nav-link" href="Classes.html">Classes</a>
+            </li>
+            <li class="nav-item px-3">
+              <a class="nav-link" href="instagram.html">Gram</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <!-- Header -->
+
+      <header class="header-kizgetdown">
+        <div class="overlay-kizgetdown">
+          <div class="container">
+            <h1 class="hero-title kizgetdown-title">FRIDAY KIZOMBA PARTY</h1>
+          </div>
+        </div>
+      </header>
+
+      <!-- Footer -->
+      <div class="footer">
+        <div class="container">
+          <h1 class="footer-title">Social media</h1>
+          <div class="text-center center-block">
+            <a href="#"
+              ><i id="social-fb" class="fab fa-facebook-square fa-2x social"></i
+            ></a>
+            <a href="#"
+              ><i id="social-yt" class="fab fa-youtube-square fa-2x social"></i
+            ></a>
+            <a href="#"
+              ><i
+                id="social-ig"
+                class="fab fa-instagram-square fa-2x social"
+              ></i
+            ></a>
+            <a href="#"
+              ><i id="social-em" class="fas fa-envelope-square fa-2x social"></i
+            ></a>
+          </div>
+        </div>
+      </div>
+
+      <!-- Optional JavaScript -->
+      <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+      <script type="text/javascript" src="/main.js"></script>
+      <script
+        src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+        integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n"
+        crossorigin="anonymous"
+      ></script>
+      <script
+        src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+        integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+        crossorigin="anonymous"
+      ></script>
+      <script
+        src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous"
+      ></script>
+    </body>
+  </head>
```

Comment:
The `<body>` is _always_ after `<head>`, not inside.

Reply:
Oh thanks for catching it, its a mistype...